README.md

```diff
@@ -1,2 +1,53 @@
 # gds-pre-commit
-An example usage for git pre-commit hooks
+This repository is here to assist GDS users in setting up pre-commit hooks that can improve the quality and security of projects hosted on GitHub.
+
+## pre-commit framework
+One of the easiest ways to get started with pre-commit hooks is by using the [pre-commit framework](https://pre-commit.com/).
+
+### Installing pre-commit
+
+To get started you can either run:
+
+`$ brew install pre-commit` (on Mac)
```

Comment:
Homebrew on Linux works for installing `pre-commit`, too. 😉

Reply:
Noted, you learn something new everyday. I missed this before I merged so I'm removing it from my next commit.

Reply:
❤️